apps/site/lib/site_web/controllers/schedule/line_api.ex

```diff
@@ -52,12 +52,25 @@ defmodule SiteWeb.ScheduleController.LineApi do
   """
   @spec predictions_and_vehicles(Plug.Conn.t(), map()) :: Plug.Conn.t()
   def predictions_and_vehicles(conn, %{"id" => route_id, "direction_id" => direction_id}) do
```

Comment:
Renaming this back to `realtime` would be nice, I think.

Reply:
@digitalcora one fine point of that, though, is we should keep the old route hooked up, so that during the transition when the old and new code are both running, it doesn't 404

Reply:
Sure; can we just redirect it, at the routing level?